packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -43,6 +43,9 @@ const useStyles = makeStyles(theme => ({
     minWidth: 260,
     maxWidth: 400,
   },
+  primary: {
+    color: theme.palette.primary.light,
```

Comment:
This has me wondering whether we should make the primary color lighter instead, since it's gonna be a bit confusing as a user of the theme to know when to use the primary and when to use the light color. Not sure if that's gonna have some broader breaking effects though 😅

Reply:
Yeah that was something that I had in mine, but opted for a granular solution using the tonalOffset and specifying it in some cases like this